apps/server/src/modules/oauth/uc/oauth.uc.ts

```diff
@@ -69,8 +70,13 @@ export class OauthUc {
 
 		const responseTokenObservable = this.httpService.post<OauthTokenResponse>(
 			tokenRequestPayload.tokenEndpoint,
-			{},
-			{ params: { ...tokenRequestPayload.tokenRequestParams } }
+			QueryString.stringify(tokenRequestPayload.tokenRequestParams),
```

Comment:
for the midterm, please not down to refactor the usecase a bit.
In my understanding, the uc should be more focussed on orchestrating steps, and less on knowing detailed information

Reply:
OK, we will have a meeting where you can maybe explain the right way to do it.